### PR TITLE
Add pull request and issue templates

### PR DESCRIPTION
**Pull request by msm-code**
This was suggested by @ITAYC0HEN. PR templates I've created are
heavily based on Cutter's (but I was also inspired by other OS
projects.

### CODE REVIEW

**Comment by msm-code** [comment]
@ITAYC0HEN would you mind taking a look at this? Please let me know if these are appropriate in your opinion.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+* Mquery version:
+* Installation method:
+  - [ ] Generic docker-compose
+  - [ ] Dev docker-compose
+  - [ ] Native (from source)
+  - [ ] Other (please explain)
+
+**Reproduction Steps**
+
+Please explain how can we reproduce your problem.
```
Comment (ITAYC0HEN) [inline .github/ISSUE_TEMPLATE/bug_report.md:17]: Seems like this line isn't commented `<!-- .... -->` while the others are. Any particular reason?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+---
+
+**Feature Category**
+
+- [ ] Correctness
+- [ ] User Interface / User Experience
+- [ ] Performance
+- [ ] Other (please explain)
+
+**Describe the problem.**
```
Comment (ITAYC0HEN) [inline .github/ISSUE_TEMPLATE/feature_request.md:14]:
```suggestion
**Describe the problem**
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+**What is the current behaviour?**
+<!-- Explain how the code works currently -->
+
+**What is the new behaviour?**
+<!-- Explain how the code works after your changes -->
+
+**Test plan**
+<!-- Explain how to test your changes -->
+
+<!-- Add in issue numbers related to this PR, if applicable -->
```
Comment (ITAYC0HEN) [inline .github/PULL_REQUEST_TEMPLATE.md:19]: It's not clear that this is **not** under "Test Plan"

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+**Environment information**
+* Mquery version:
```
Comment (nazywam) [inline .github/ISSUE_TEMPLATE/bug_report.md:8]: Would we be interested in the ursadb version as well?

**Pull request merged**